frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -2077,6 +2078,27 @@ Component that allows to receive contributions from other plugins for the `conso
 
 
 
+---
+
+## `NamespaceBar`
+
+### Summary 
+
+A component that renders a namespace dropdown menu.
+
+
+
+
+### Parameters
+
+| Parameter Name | Description |
+| -------------- | ----------- |
+| `isDisabled` | (optional) A boolean flag that disables the dropdown if set to true |
```

Comment:
Do we need this flag? It's unclear to me when I would need to use that.

If we keep it, we need to describe whether `children` are rendered when the bar is disabled.

Reply:
I'm not sure. It's heavily used in the dev-console package, but I don't fully understand why they've used it where they have.

I updated the description as suggested. Let me know if you'd rather just not expose it.